### PR TITLE
Fixed chat renders when using StackedAreaChart without area and with brush and zoom

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Chart renders incorrectly when using StackedAreaChart without area and with brush and zoom

Issue Number: [21](https://github.com/reaviz/reaviz/issues/21)

## What is the new behavior?
Fixed chat renders when using StackedAreaChart without area and with brush and zoom

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
https://user-images.githubusercontent.com/1290843/209805762-ebef457b-10bd-4984-b077-0595dd985cc1.mov

### CODE REVIEW

File: src/AreaChart/AreaSeries/Line.tsx [hunk 1]
```diff
@@ -148,17 +148,12 @@ export const Line: FC<Partial<LineProps>> = ({
   const enter = useMemo(() => {
     const linePath = getLinePath(coords);
 
-    let strokeDasharray = '';
```
Comment (amcdnl) [inline src/AreaChart/AreaSeries/Line.tsx:151]: Can you test if the line chart works when you transition it? AKA default value then change the value. I think thats what this code is for.

**Review by amcdnl — CHANGES_REQUESTED** [review]
see feedback

File: src/AreaChart/AreaSeries/Line.tsx (hunk 1 above)
Comment (evgenoid) [reply src/AreaChart/AreaSeries/Line.tsx:151]: I've checked all LineChart stories with these changes and without - and I don't see any difference

Comment (amcdnl) [reply src/AreaChart/AreaSeries/Line.tsx:151]: Ya - Its not animating - it just jumps to the next point. Watch the video here, the first one animates to the next point ( master ) the last one jumps to it ( your branch ).

https://app.screencast.com/hMr9yN5vGWIem

**Comment by amcdnl** [comment]
@evgenoid - any updates here?

**Comment by evgenoid** [comment]
> @evgenoid - any updates here?

No. Need to go back and look in details

**Comment by amcdnl** [comment]
ok going to close for now then.

**Pull request closed without merging**